contracts/ibc-denom-resolver/src/error.rs

```diff
@@ -24,4 +24,7 @@ pub enum ContractError {
 
     #[error("The length of receivers must be same to the length of routes")]
     InvalidReceiversLength,
+
+    #[error("Route is designated only for terminal")]
+    InvalidRoutes,
```

Comment:
no need for creating this error because the case of memo being `None` is only when routes are empty vec

Reply:
no, as the title says "when only terminal route exists " (https://github.com/UnUniFi/packet-forward-contract/issues/4), none is created even if routes aren't empty.
That's why I made another error type there

Reply:
ah I see...